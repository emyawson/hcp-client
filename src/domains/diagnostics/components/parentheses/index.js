export { ParenthesesWrapper } from './parentheses.component';
