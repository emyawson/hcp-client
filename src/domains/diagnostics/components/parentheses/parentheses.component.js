import React from 'react';

export const ParenthesesWrapper = ({ text }) => <span>({text})</span>;
