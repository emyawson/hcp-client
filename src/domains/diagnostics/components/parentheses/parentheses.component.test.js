import React from 'react';
import { shallow } from 'enzyme';

import { ParenthesesWrapper } from './parentheses.component';

describe('Parentheses component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ParenthesesWrapper text="in parentheses" />);
    expect(wrapper).toMatchSnapshot();
  });
});
