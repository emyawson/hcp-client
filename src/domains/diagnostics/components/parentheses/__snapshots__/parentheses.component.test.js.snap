// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parentheses component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    text="in parentheses"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "(",
        "in parentheses",
        ")",
      ],
    },
    "ref": null,
    "rendered": Array [
      "(",
      "in parentheses",
      ")",
    ],
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "(",
          "in parentheses",
          ")",
        ],
      },
      "ref": null,
      "rendered": Array [
        "(",
        "in parentheses",
        ")",
      ],
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
